Packs/PhishTank/Integrations/PhishTankV2/PhishTankV2.py

```diff
@@ -0,0 +1,307 @@
+                                of the last reload.
+    """
+    data = demisto.getIntegrationContext()
+    status = "PhishTankV2 Database Status\n"
+    data_was_not_reloaded_yet = data == dict()
+    if data_was_not_reloaded_yet:
+        status += "Database not loaded.\n"
+    else:
+        last_load = datetime.utcfromtimestamp(data["timestamp"] / 1000.0).strftime("%a %b %d %Y %H:%M:%S (UTC)")
+        status += f'Total **{len(data["list"].keys())}** URLs loaded.\n' \
+                  f'Last Load time **{last_load}**\n'
+    return CommandResults(readable_output=status)
+
+
+def reload(client: Client) -> dict:
+    """
+    This function is responsible for:
+     1. request a csv file from PhishTank API (calling to client.get_http_request)
+     2. parsing an API response and saving all relevant information into a dictionary
+
+    Args:
+        client:
+        (Client) : client to use in the PhisTankV2 integration.
+
+    Returns:
+        dictionary of parsed http response. Each url is a key and his values are:
+            "id,submission_time,verified,verification_time,online,target"
+    """
+    response = client.get_http_request(RELOAD_DATA_URL_SUFFIX)
+    response_is_empty = not response
+    if response_is_empty:
+        return dict()
+    response = response.splitlines()
+    parsed_response = {}
+    columns = response[0].strip().split(",")  # get csv headers
+    for index, line in list(enumerate(response))[1:]:
+        line = line.split(",")
+        line = parse_response_line(line, index, response)
+        invalid_parsed_line = line is None
+        if invalid_parsed_line:
+            continue
+        url = remove_last_slash(line[columns.index("url")])
+        if url:
+            parsed_response[url] = {
+                "phish_id": line[columns.index("phish_id")].strip(),
+                "submission_time": line[columns.index("submission_time")].strip(),
+                "verified": line[columns.index("verified")].strip(),
+                "verification_time": line[columns.index("verification_time")].strip(),
+                "online": line[columns.index("online")].strip(),
+                "target": line[columns.index("target")].strip(),
+            }
+    return parsed_response
+
+
+def parse_response_line(current_line, index, response):
+    """
+    This function checks if current line is a valid line.
+    note: there is a specific line in PhishTank csv response that is broken into 2 following lines. In this case,
+            those 2 lines are concatenate into one complete line.
+    Args:
+        current_line (str): current response's line to be parsed
+        index (int) : current line's index
+        response (list) : list of PhishTank csv response
+
+    Returns: line (str): the parsed line
+
+    """
+    current_line_length = len(current_line)
+    # RESPONSE_LINE_LENGTH is the number of valid columns in csv
+    line_length_invalid = current_line_length < RESPONSE_LINE_LENGTH
```

Comment:
let's rename, should mean that the line is broken  if I understood correctly

Reply:
Done